Library/include/CSP/Systems/EventTicketing/EventTicketing.h

```diff
@@ -168,6 +219,9 @@ typedef std::function<void(const TicketedEventResult& Result)> TicketedEventResu
 // @brief Callback providing a ticketed event collection result.
 typedef std::function<void(const TicketedEventCollectionResult& Result)> TicketedEventCollectionResultCallback;
 
+// @brief Callback providing a ticketed event result.
+typedef std::function<void(const EventTicketResult& Result)> EventTicketResultCallback;
```

Comment:
`SubmitEventTicketResultCallback` might be clearer?

Reply:
`EventTicket` is a mapping of the DTO for the ticket resource and this is a result class that holds an `EventTicket`. It's ppossible this could be used again in another operation, e.g. `GetTickets` so I don't think we should name it specifically for the operation that uses it. Similarly we call the result for `Create/Update/GetTicketedEvents` `TicketedEventResult`.

Reply:
Ok that makes sense, thanks!